js/samples/quickstart-csharp/Controllers/HomeController.cs

```diff
@@ -1,21 +1,19 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
 using QuickstartSampleWebApp.Models;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.Identity.Client;
 
 namespace QuickstartSampleWebApp.Controllers
 {
-	public class HomeController : Controller
-	{
+    public class HomeController : Controller
+    {
 		private readonly string TenantId;     // Azure subscription TenantId
 		private readonly string ClientId;     // Azure AD ApplicationId
 		private readonly string ClientSecret; // Azure AD Application Service Principal password
 		private readonly string Subdomain;    // Immersive Reader resource subdomain (resource 'Name' if the resource was created in the Azure portal, or 'CustomSubDomain' option if the resource was created with Azure CLI Powershell. Check the Azure portal for the subdomain on the Endpoint in the resource Overview page, for example, 'https://[SUBDOMAIN].cognitiveservices.azure.com/')
 
+		IConfidentialClientApplication app;
```

Comment:
create a property for this to do the lazy initialize: ConfidentialClientApplication

Reply:
done